### ISSUE TITLE
Plan mailing list archives management 

### ISSUE THREAD

**Issue by iayanpahwa** [ILUGD/volunteers-tasks#2]

**Comment by hellozee** [comment]
I wasn't able to contact the person who handled the mailing lists, or else I would have enabled that way back. I will try to contact Shyam and see if I can do something or not. Or else we could switch to a new mailing list.

**Comment by miphilomath** [comment]
I agree. We can move to a new service provider, https://www.freelists.org/. Also, we need to check how subscribers from old lists can be migrated.

**Comment by xeon-zolt** [comment]
#7

**Comment by tm9k1** [comment]
If freelists.org works fine for everyone I'd like to step up and manage a mailing list on behalf of ILUGD at freelists.org . Lets take this one up in the Telegram group and discuss with all hands on deck.

**Comment by iayanpahwa** [comment]
@hellozee will export all the mailing list archives starting January 2020 and dump it on a new repo on github. @rajudev has archives since 2017. Let's put it on a new github repo named "Mailing list archives" under directory structure representing year.
